src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -103,6 +107,17 @@ public Person toModelType() throws IllegalValueException {
         }
         final Email modelEmail = new Email(email);
 
+        if (favourite == null) {
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT,
+                    Favourite.class.getSimpleName()));
+        }
+        final Favourite modelFavourite;
+        if (favourite.equals("Favourited")) {
+            modelFavourite = new Favourite(true);
+        } else {
+            modelFavourite = new Favourite(false);
+        }
```

Comment:
Would it be possible to have something like `Favourite.isFavourited(favourite)` similar to how the other classes `Email`, `Address` in the same `toModelType()` method of `Person` class did it? 

That way, all functionalities of `Favourite` are contained within the class itself

Reply:
Resolved!